support/cas-server-support-cassandra-ticket-registry/src/main/java/org/apereo/cas/ticket/registry/CassandraTicketRegistryDao.java

```diff
@@ -0,0 +1,29 @@
+package org.apereo.cas.ticket.registry;
+
+import org.apereo.cas.ticket.TicketGrantingTicket;
+
+import java.util.stream.Stream;
+
+/**
+ * @author David Rodriguez
+ *
+ * @since 5.1.0
+ */
+public interface CassandraTicketRegistryDao {
```

Comment:
I am not sure I understand the role of this one too. What does it do, and how is it different from the normal ticket registry API, so to speak?